docs/en/ingest-management/fleet/fleet-settings.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[fleet-settings]]
+= {fleet} settings
+
+beta[]
+
+Configure {fleet} settings in {kib} to apply global settings to all {agent}s
+enrolled in {fleet}:
+
+. In {kib}, open the main menu, then click *Management > {fleet}*.
+
+. Click *{fleet} settings*
++
+--
+[cols="2*<a"]
+|===
+|
+[id="fleet-server-hosts-setting"]
+*{fleet-server} hosts*
+
+| The URLs your {agent}s will use to connect to a {fleet-server}. This setting
+is required. On self-managed clusters, you must specify one or more URLs. On
+{ecloud}, this field is populated automatically.
+
+//QUESTION: What is valid here? URL implies something like http://localhost:8220.
+//Is the scheme and port required or does the setting use defaults if their
+//not specified?
+
+//QUESTION: What do we need to say about IPv6 addresses?
+
+By default, {fleet-server} is exposed on the following ports:
+
+`8220`::
+Default {fleet-server} port for self-managed clusters
+
+`443`::
+Default {fleet-server} port for {ecloud}
+
+//QUESTION: Is this true? Is 443 always the default on cloud?
+
+Specify multiple URLs to scale out your deployment and provide automatic
+failover.
+
+If multiple URLs exist, {fleet} shows the first provided URL for enrollment
+purposes. Enrolled {agent}s will connect to the URLs in round robin order until
+they connect successfully. 
+
+//QUESTION: I'm assuming users can use any Fleet Server for enrollment though?
```

Comment:
I have not attempted multiple Fleet server URL usage yet (with any success), I think having the docs forward looking (to 7.14) is a-ok for now, but I can't make statements based on testing I've done.   I think it is correct that any Fleet Server could be used.  :/

Reply:
I think you're right. I will keep the statement and we can remove it later if necessary.